### ISSUE TITLE
Consider changing name to samething like "uBlock Defender"

### ISSUE THREAD

**Issue by sbstn87** [LiCybora/NanoDefenderFirefox#188]
Hi,

after what has recently happened and cutting ties with the upstream NanoDefender due to the ownership change, you should consider changing name of the project to something like "uBlock Defender" or "uBlock Protector", as at the moment it is only the uBlock Origin companion extension.

**Comment by LiCybora** [comment]
I agree to rename, but I am confuse should I just use the term `uBlock` due to name confusion to another scam (uBlock-LLC). Including the name uBO or too similar naming may also cause their contributors trouble, as this project is not related to them as well.

I am also planning to open as a new repository. The new coward owner just clone everything from old repository and still linking me as their "official port", which is fake news at all.

**Comment by rari-teh** [comment]
Maybe something like Cybora Defender? It could also be a more direct name like Antiantiadblock but that borders on the silly

Thank you so much for maintaining this fork, you’re a legend

**Comment by gabri25ele** [comment]
uBlock Defender like me!

**Comment by LiCybora** [comment]
It is not good to use a name include someone else brand. Users who seldom read and pay attention might easily misinterpret as it is related to uBO, which might give their contributors trouble.

I am thinking like "Pico Defender", a continuation after **μ**Block (old name of uBlock) and **nano** for unit prefix. The downside is people may not understand what it is for just from the naming.

Decision is not final yet, feel free to present your idea.

**Comment by gabri25ele** [comment]
yes, it's true... I like "ublock" in the name to a connextion to the "origin" but I understand it's not good. it might even look like a counterfeit... 
and If you will use the "origin"? I don't know...
anyway, the update will come automatically as update of nano defender 15.0.0.206 or need to be installed by new?

**Comment by LiCybora** [comment]
New install if you are using the "Nano Defender Pro" on GitHub release. Installing from AMO will receive direct update, but I will provide sufficient information to notify user. Next nano defender update will show the message at first install/update.

But this repository will most likely not using anymore. I will open a new repository instead and archive this repository.

**Comment by sbstn87** [comment]
uDefender or uProtector?

**Comment by gabri25ele** [comment]
**uDefender**! I like it!

**Comment by LiCybora** [comment]
uDefender sounds good. If no objection today, I will take this name

**Comment by thetinsmith** [comment]
On 10/16/2020 7:02 PM, LiCybora wrote:

 > uDefender sounds good. If no objection today, I will take this name



The choice is yours of course. But I like it.


-- 



   David

**Comment by Peacock365** [comment]
Whatever new name you choose, also change the logo of the extension. The Nano brand is heavily damaged, it is in your best interest to remove all reference to it once the extension is renamed and rebranded. Also rename the source repositories, none of them should carry the Nano name. Once that is done, nothing speaks against this extension thriving again in the future on Firefox.

**Comment by gwarser** [comment]
Logo already selected https://github.com/LiCybora/NanoCoreFirefox/issues/36

![logo](https://user-images.githubusercontent.com/454772/96340014-29dd2700-1066-11eb-9e83-1257897c6316.png)

**Comment by ExE-Boss** [comment]
I’m personally in favour of **PicoDefender**.
